### DISCUSSION TITLE
The keynote of the pattern changes

### DISCUSSION THREAD

**Discussion by maxtribe** [bangcorrupt/hacktribe#45] [Q&A] [answered]
Hello people

Does anyone of you have any idea why the whole pattern changes from the pitch, for example when I change from the root C to the root F. This was not the case with the previous firmware.

Thanks for the help in advance

Kind regards Max

**Comment by BKLronin** [comment]
Can confirm, I got the same.

**Comment by bangcorrupt** [comment, marked as answer]
Did you start with 'Initialize Pattern'?  The new default init pattern has 'Scale Mode' on for all parts, turn it off and the pitch won't change. 

You can use `scripts/e2s-init-pat.py` when patching the firmware to set a custom init pattern of your choice.  

Also, see #24.
